src/components/partnership/partnership.service.ts

```diff
@@ -383,21 +383,82 @@ export class PartnershipService {
       throw new NotFoundException('could not find Partnership');
     }
 
+    const projectMapPartner = this.db
+      .query()
+      .match(matchSession(session, { withAclRead: 'canReadPartnerships' }))
+      .match([node('partnership', 'Partnership', { active: true, id })]);
+    projectMapPartner.optionalMatch([
+      node('requestingUser'),
+      relation('in', '', 'member', { active: true }),
+      node('', 'SecurityGroup', { active: true }),
+      relation('out', '', 'permission', { active: true }),
+      node('projectMouStartPerm', 'Permission', {
+        property: 'mouStart',
+        active: true,
+        read: true,
+      }),
+      relation('out', '', 'baseNode', { active: true }),
+      node('project'),
+      relation('out', '', 'mouStart'),
+      node('projectMouStart', 'Property', {
+        active: true,
+      }),
+    ]);
+    projectMapPartner.optionalMatch([
+      node('requestingUser'),
+      relation('in', '', 'member', { active: true }),
+      node('', 'SecurityGroup', { active: true }),
+      relation('out', '', 'permission', { active: true }),
+      node('projectMouEndPerm', 'Permission', {
+        property: 'mouEnd',
+        active: true,
+        read: true,
+      }),
+      relation('out', '', 'baseNode', { active: true }),
+      node('project'),
+      relation('out', '', 'mouEnd'),
+      node('projectMouEnd', 'Property', {
+        active: true,
+      }),
+    ]);
+
+    projectMapPartner.return({
+      projectMouStart: [{ value: 'projectMouStart' }],
+      projectMouStartPerm: [
+        {
+          read: 'canReadProjectMouStart',
+          edit: 'canEditProjectMouStart',
+        },
+      ],
+      projectMouEnd: [{ value: 'projectMouEnd' }],
+      projectMouEndPerm: [
+        {
+          read: 'canReadProjectMouEnd',
+          edit: 'canEditProjectMouEnd',
+        },
+      ],
+    });
+
+    const readProject = await projectMapPartner.first();
+
     let mouStart = null;
     let mouEnd = null;
 
     // if user has access to project mou and there is no partnership override
-    if (result.mouStart.canRead || result.mouStartOverride.canRead) {
-      mouStart = result.mouStartOverride.value ?? result.mouStart.value;
+    if (
+      readProject?.canReadProjectMouStart ||
+      result.mouStartOverride.canRead
+    ) {
+      mouStart = result.mouStartOverride.value ?? readProject?.projectMouStart;
```

Comment:
this is good, just make sure the value comes from call the project readOne function.